### PR TITLE
Permit URL advertised in the IS-04 controls array to be extended on a …

### PR DESCRIPTION
**Pull request by btrost-harmonicinc**
These changes are as we discussed in yesterday's Audio Channel Mapping conference call.

@grmph555, I believe this phrasing addresses your concern about being able to specify a single API endpoint for all Devices.

### CODE REVIEW

File: docs/2.0. APIs.md [hunk 1]
```diff
@@ -28,7 +28,13 @@ All NMOS APIs MUST use a path in the following format. Other HTTP resources MAY
 http(s)://<ip address or hostname>:<port>/x-nmos/<api type>/<api version>/
 ```
 
-At each level of the API from the base resource upwards, the response SHOULD include a JSON format array of the child resources available from that point.
+Nodes MAY extend the path for the Audio Channel Mapping API with a suffix. A Node implementation might choose to extend the path so that Devices on the Node have separate Audio Channel Map APIs without having to use a different port for each Device's API. Such a path SHALL be formatted as
+
+```
+http(s)://<ip address or hostname>:<port>/x-nmos/<api type>/<api version>/<device selector>/
```
Comment (btrost-harmonicinc) [inline docs/2.0. APIs.md:34]: I used the word "selector" because "id" might imply the Device's UUID, which seems like an unnecessary restriction.

Do we need to constrain the form of the selector? In particular, would slashes within the selector present a problem for control systems?

Also, as phrased, Nodes may use a suffix on some Devices and not others. While strange, would this present a problem?

File: docs/2.0. APIs.md [hunk 2]
```diff
@@ -28,7 +28,13 @@ All NMOS APIs MUST use a path in the following format. Other HTTP resources MAY
...
+```
+
+At each level of the API from the base resource onwards, the response SHOULD include a JSON format array of the child resources available from that point.
```
Comment (btrost-harmonicinc) [inline docs/2.0. APIs.md:37]: This change is unrelated to the URL change. I found "upwards" confusing ("I'm looking down, Pooh").

**Comment by andrewbonney** [comment]
Likely unavoidable, but this will be a bit of a 'special case' as far as the test suite is concerned given that it won't be possible to determine this via the RAML. It might be worth adding a specific issue against the test suite so we can consider how this might be resolved.

File: docs/2.0. APIs.md (hunk 1 above)
Comment (chris-gil) [reply docs/2.0. APIs.md:34]: The variation should not matter - the controller would use the href in the control array as is.  If it does contain a suffix then that would be used as the base path for IS-08 methods.

**Review by simonrankine — APPROVED** [review]
This looks good to me
